resources/views/components/game/leaderboards-listing/index.blade.php

```diff
@@ -8,8 +8,13 @@
     @if (!$game->visibleLeaderboards()->exists())
         <x-game.leaderboards-listing.empty-state :game="$game" />
     @else
-        <x-game.leaderboards-listing.leaderboards-list
-            :gameLeaderboards="$game->visibleLeaderboards->sortBy('DisplayOrder')"
-        />
+        @php
+            $gameLeaderboards = $game->visibleLeaderboards()
+                ->withTopEntry()
+                ->orderBy('DisplayOrder')
+                ->get();
+        @endphp
+
+        <x-game.leaderboards-listing.leaderboards-list :$gameLeaderboards />
```

Comment:
This seems to work, but shouldn't it be `:gameLeaderboards="$gameLeaderboards"`?

Reply:
`:$variableName` is Blade shorthand for `:variableName="$variableName"`. I only learned about this feature very recently.